src/link_prober/LinkProber.cpp

```diff
@@ -394,6 +396,14 @@ void LinkProber::handleTimeout(boost::system::error_code errorCode)
     if (mTxSeqNo != mRxSelfSeqNo && mTxSeqNo != mRxPeerSeqNo) {
         // post unknown event
         mLinkProberStateMachine.postLinkProberStateEvent(LinkProberStateMachine::getIcmpUnknownEvent());
+        mContinuousIcmpUnknownEventCount++; 
+        mIcmpUnknownEventCount++;
+
+        MUXLOGINFO(boost::format("%s: continusous ICMP Unknown event count: %d, pck loss rate since initialization: %.2f %%") %
+            mMuxPortConfig.getPortName() %
+            mContinuousIcmpUnknownEventCount %
+            (static_cast<double_t>(mIcmpUnknownEventCount) / mIcmpPacketCount * 100) 
```

Comment:
Will this event be too chatty when the peer is not there?

Reply:
Yes, you're right, it can be too chatty. Now I think about it we can: 
* Simply keep incrementing the interval of this log message in this case.
* Or, create a DB table for ICMP packet loss info collection, and use command line "show" to pull this info when needed? This will require changes in orchagent & sonic-utilities as well. Can we discuss offline, just in case I overkill this problem?